assets/js/hooks/useBreakpoint.test.js

```diff
@@ -50,18 +64,11 @@ describe( 'useBreakpoint', () => {
 			1281,
 			'xlarge',
 		],
-	] )( '%s', async ( _, args, expected ) => {
+	] )( '%s', async ( _, viewportWidth, expected ) => {
 		let result;
 		await act( async () => {
 			( { result } = await renderHook( () => {
-				Object.defineProperty(
-					global.window.document.documentElement,
-					'clientWidth',
-					{
-						value: args,
-						configurable: true,
-					}
-				);
+				setViewportWidth( viewportWidth );
```

Comment:
This looks like a nice improvement – is it related to the other changes?

Reply:
Yeah, I extracted `setViewportWidth` from here to reuse in `assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Overview.test.js` elsewhere in this PR.